File: 0266-palindrome-permutation/0266-palindrome-permutation.cpp
class Solution {
public:
    bool canPermutePalindrome(string s) {
        map<char,int> hashMap; 
        int final_cnt = 0; 

        for(char& c : s){
            if(hashMap.count(c)) hashMap[c]++; 
            else hashMap[c] = 1; 
        }

        for(auto& it : hashMap){
            if(it.second % 2 == 1) final_cnt++; 
        }

        return final_cnt > 1 ? false : true; 
    }
};

File: 0266-palindrome-permutation/0266-palindrome-permutation.java
class Solution {
    public boolean canPermutePalindrome(String s) {
        Map<Character,Integer> hashMap = new HashMap<>(); 
        for(char c : s.toCharArray()) hashMap.merge(c,1,Integer::sum); 

        int count = 0; 
        for(var e : hashMap.entrySet()){
            if(e.getValue() % 2 != 0) count++;
            if(count > 1) return false; 
        }
        return true; 
    }
}